<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PickerDialog</class>
 <widget class="QDialog" name="PickerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="horizontalWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="viewLayout">
          <item>
           <widget class="QTreeView" name="filesystemView">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>120</height>
             </size>
            </property>
            <property name="rootIsDecorated">
             <bool>true</bool>
            </property>
            <property name="uniformRowHeights">
             <bool>true</bool>
            </property>
            <property name="sortingEnabled">
             <bool>false</bool>
            </property>
            <property name="headerHidden">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="sidebar">
          <item>
           <widget class="PixmapGraphicsView" name="graphicsView">
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
            </property>
            <property name="viewportUpdateMode">
             <enum>QGraphicsView::ViewportUpdateMode::FullViewportUpdate</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PickerChildWidget" name="wrappedWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Apply|QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Help|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PickerChildWidget</class>
   <extends>QWidget</extends>
   <header>ui/picker/picker_child_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PixmapGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>ui/viewer/rpg_graphics_view.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
